pkg/runner/simple.go

```diff
@@ -93,11 +93,17 @@ func (r *SimpleRunner) Execute(req *ffuf.Request) (ffuf.Response, error) {
 	if err != nil {
 		return ffuf.Response{}, err
 	}
+
 	// Add user agent string if not defined
 	if _, ok := req.Headers["User-Agent"]; !ok {
-		req.Headers["User-Agent"] = fmt.Sprintf("%s v%s", "Fuzz Faster U Fool", ffuf.VERSION)
+		if _, ok := req.Headers["User-agent"]; !ok {
+			req.Headers["User-Agent"] = fmt.Sprintf("%s v%s", "Fuzz Faster U Fool", ffuf.VERSION)
+		} else { 
+			req.Headers["User-Agent"] = req.Headers["User-agent"]
+		}
 	}
-	// Handle Go http.Request special cases
+	
+        // Handle Go http.Request special cases
```

Comment:
These whitespace and comment changes should be reverted

Reply:
reverted